dace/codegen/compiled_sdfg.py

```diff
@@ -273,6 +278,241 @@ def __call__(self, *args, **kwargs):
                 {aname: arg
                  for aname, arg in zip(self.argnames, args)})
 
+        if vscode.is_available() and vscode.sdfg_edit():
+            response = vscode.send_bp_recv({'type': 'sdfgEditMode'})
+            mode = response['mode']
+            compiledSdfg = self
+
+            while mode != 'continue':
+                # unload lib as the sdfg might change
+                if compiledSdfg._lib.is_loaded():
+                    compiledSdfg._lib.unload()
+
+                # Create a deepcopy as the args could change after an execution
+                initital_args = copy.deepcopy(kwargs)
+                compiledSdfg._init = None
+                argtuple, initargtuple = compiledSdfg._construct_args(
+                    initital_args)
+
+                if mode == 'run':
+                    # Runs the SDFG program once.
+                    # Prints all arguments and results.
+                    print('Arguments:')
+                    for arg, arr in initital_args.items():
+                        print('- ', arg, ': ', arr)
```

Comment:
```suggestion
```

Reply:
I suggest keeping these print statements. They could give a good intuition if the input arguments and output values seem fine.